packages/flutter_tools/ide_templates/intellij/.idea/runConfigurations/manual_tests___density.xml.copy.tmpl

```diff
@@ -0,0 +1,6 @@
+<component name="ProjectRunConfigurationManager">
```

Comment:
Does this really belong the PR?

Reply:
Yes.  It adds the run configuration for the manual test density app for IntelliJ.